index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <script src="index.js" defer></script>
+  <link rel="stylesheet" href="style.css">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Calculator</title>
+</head>
+<body>
+  <div class="container">
+    <div class="display">
+  
+    </div>
+    <div class="btn-container">
+      <div class="ctn-1">
+        <button id="clear" class="btn1">AC</button>
```

Comment:
`btn1` is a not a well describing class name. Later I see `btn` and `btn0` which is not very clear.
I also see `btn operator` -> that's a better approch to give the "btn" class only the basic button stylings and then differenciate them by type like `btn operator`, `btn number`, `btn calculate`

I also like to define a class per component. Right now you only have the calculator component, but later you probably will have several components in one application.
If you introduce a class `calculatr` for the container in class 12, then you can define your styles more specific:
e.g. `.calculator .btn.operator` to make sure that you only style the operator buttons for the calculator component and not accidentally the buttons of a different component which happens to use the same class names.